packages/elements/src/styles/components/menu.scss

```diff
@@ -94,8 +98,28 @@ $nav-dimensions: 66px;
       height: 42px;
     }
   }
+}
+
+.nav-bar.is-desktop {
+  background-color: $grey-lightest;
+  border-right: 2px $grey-light solid;
+
+  li {
+    border: $grey-lightest 3px solid;
+    color: $grey;
+  }
 
-  &.is-desktop {
-    display: none;
+  li.is-active,
+  :hover {
+    color: $black;
+  }
+
+  li.is-active {
+    border-bottom: $black 3px solid;
+
+    @media screen and (min-width: 768px) {
```

Comment:
We've built mixins for media queries in `packages/marketplace/src/styles/base/screen-size.scss`.
Please use them from there